### ISSUE TITLE
VcdApiClient.get returns inconsistent responses in 9.1 vs 9.5

### ISSUE THREAD

**Issue by Draper1** [vmware/vcd-ext-sdk#71]
**Describe the bug**
Currently using VcdApiClient.get to determine a list of API Versions available however, in response the format is different making handling data more difficult.

**To Reproduce**
Using the following line will return a different JSON Body:
`return this.vcdApi.get('api/versions').pipe(map resp=> {console.log(resp)})`

In 9.1 we see the format of the body as:
```{
  "name" : "{http://www.vmware.com/vcloud/versions}SupportedVersions",
  "declaredType" : "com.vmware.vcloud.api.rest.schema.versioning.SupportedVersionsType",
  "scope" : "javax.xml.bind.JAXBElement$GlobalScope",
  "value" : {
    "versionInfo" : [ {
      "version" : "5.5",
      "loginUrl" : "https://<vcd-cell-redacted>/api/sessions",
      "mediaTypeMapping" : [ {
        "mediaType" : "application/vnd.vmware.vcloud.error+xml",
        "complexTypeName" : "ErrorType",
        "schemaLocation" : "http://<vcd-cell-redacted>/api/v1.5/schema/common.xsd",
        "any" : [ ],
        "otherAttributes" : { }
      },
```

in 9.5, we see the following:
```
{
  "versionInfo" : [ {
    "version" : "20.0",
    "loginUrl" : "https://<vcd-cell-redacted>/api/sessions",
    "mediaTypeMapping" : [ ],
    "any" : [ ],
    "deprecated" : true,
    "otherAttributes" : { }
  },
```
No type information is displayed in 9.5.

When handling the response and reading it's length, we need to handle 9.1 with a `value` entry and without for 9. This also applies for reading any of response body:

9.1 - `resp['body']['value']['versionInfo'].length`
9.5 - `resp['body']['versionInfo'].length`

**Expected behavior**
Bodies should be consistent since we're using the same SDK for both versions of vCloud Director.

**Desktop (please complete the following information):**
 - OS: [Mac OSX]
 - Browser [Chrome]
 - Version [30.0 and 31.0]

**Comment by jmoroski** [comment]
Can you confirm that this is specific to the /version calls and not to the bulk of API calls in 9.1? There are different object mappers in place for that end point, and we will need to work around that extra wrapping in the SDK layer.

**Comment by jmoroski** [comment]
Closed in pull request #75
